_includes/domain-search.html

```diff
@@ -0,0 +1,100 @@
+{% comment %}
+TODO: Domain search description
+{% endcomment %}
+
+<script>
+    function checkDomainAvailability() {
+        // TODO: update baseUrl to stable when Cloudflare migrates their data 11/15
+        const baseUrl = "https://getgov-staging.app.cloud.gov/api/v1/available/"
```

Comment:
**non blocking comment** I think this url would be be better in some constants file, or even as an environment variable added in package.json. This would be in case we end up adding more connections between the two sites (which we will), we wouldn't have to update the external domain portion of the url in multiple locations. That being said, you have met the AC of your ticket and this could just be changed on the next ticket that updates this to point to staging, seems reasonable

Reply:
This is a great call! I have added it as [a comment](https://github.com/cisagov/manage.get.gov/issues/1306#issuecomment-1803956206) in the ticket we'll be changing this in :)